packages/employer-api/src/controllers/event.controller.ts

```diff
@@ -57,6 +58,21 @@ export const submission = async (req: any, res: express.Response) => {
                 )
                 if (addResult === 1) {
```

Comment:
Does this check now always fail? Since claimService.addServiceProviderClaim now returns an array of form [catchmentno, id], rather than the number of records updated?

Reply:
Fixed function back to its original returning values. This check would now be functional.